app/views/specialist_document/show.html.erb

```diff
@@ -10,7 +10,10 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds responsive-top-margin">
-    <%= render "govuk_publishing_components/components/title", @content_item.title_and_context %>
+    <%= render "govuk_publishing_components/components/title", {
```

Comment:
I don't know if this is covered in a later commit, but we don't want to be using the title component - we want to be using the [heading component[(https://components.publishing.service.gov.uk/component-guide/heading) instead. (title is being removed very soon).